substdio.h

```diff
@@ -24,6 +24,10 @@ extern int substdio_bputs();
 extern int substdio_putsflush();
 
 extern ssize_t substdio_get(substdio *s, char *buf, size_t len);
+#include "deprecated.h"
```

Comment:
Please move this header to the top where the other include is.

Reply:
I was aiming to follow the existing style, where (so far, anyway) we include the deprecation header just before it’s first used. I’m okay with changing the style. Should I also fix the other usages?

Reply:
Uh, yes please. Why did I do this?

Reply:
It was probably me, that’s why :-)

Reply:
I've moved them all up, inside the include guards and, since it's the current style, after system headers (if any). #183 will want to move them slightly further up.